devnet/src/mpc.rs

```diff
@@ -199,6 +208,70 @@ impl UpdateMpcNetworkCmd {
     }
 }
 
+#[derive(Clone, Deserialize)]
+pub struct StaticWebData {
+    pub near_signer_public_key: near_crypto::PublicKey,
+    pub near_p2p_public_key: near_crypto::PublicKey,
+    pub near_responder_public_keys: Vec<near_crypto::PublicKey>,
+    pub _tee_participant_info: Option<TeeParticipantInfo>,
+}
```

Comment:
Hmm, I wonder if we should consider reusing the existing `StaticWebData` struct from `node/src/web.rs` to prevent this from going stale. What do you think?

Reply:
Issue: https://github.com/near/mpc/issues/719
and PR: https://github.com/near/mpc/pull/720